beaconChain/constants/constants.ts

```diff
@@ -45,3 +47,8 @@ export const MAX_CASPER_SLASHINGS = 2 ** 4; // 16
 export const MAX_ATTESTATIONS = 2 ** 7; // 128
 export const MAX_DEPOSITS = 2 ** 4; // 16
 export const MAX_EXITS = 2 ** 4; // 16
+
+// Smart contract addresses
+export const MAINNET_DEPOSIT_ADDRESS = "0x0"; // Address for teh deposit function on ethereum.
+export const DEPOSIT_CONTRACT_BLOCK_NUMBER = 5222222;
```

Comment:
Can you explain the use case for this?

Reply:
It actually should be stubbed to "tbd". But it's the block at which the contract is deployed, so we can start looking for receipts at that address, and then work forward

Reply:
Isn't that what the ChainStart event is for? As I understand it deposits made after are still valid so I don't see the point in differentiating those made before and from those made after.